LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2018 Túlio Ornelas (ornelas.tulio@gmail.com)
+Copyright (c) 2019 Túlio Ornelas (ornelas.tulio@gmail.com)
```

Comment:
The year in the copyright notice represents the year of publication. Basically how far back you are claiming copyright. There's no need to update it.